metrics/pcp/pcp.go

```diff
@@ -0,0 +1,129 @@
+package pcp
+
+import (
+	"github.com/go-kit/kit/metrics"
+	"github.com/performancecopilot/speed"
+)
+
+// Reporter encapsulates a speed client
+type Reporter struct {
+	c *speed.PCPClient
+}
+
+// NewReporter creates a new reporter instance
+func NewReporter(appname string) *Reporter {
+	c, err := speed.NewPCPClient(appname)
+	if err != nil {
+		panic(err)
+	}
+	return &Reporter{c}
+}
+
+// Start starts reporting currently registered metrics to the backend
```

Comment:
1. Please use proper punctuation (further comments of this type elided).
2. What does "backend" mean?